utils.go

```diff
@@ -54,7 +54,7 @@ func AssureExists(filePath string) error {
 	if !exists {
 		err = os.MkdirAll(path, os.ModePerm)
 		if err != nil {
-			return fmt.Errorf("Couldn't create file: %s", filePath)
+			return fmt.Errorf("Couldn't create path: %s", path)
```

Comment:
This change kind of slipped in and is unrelated to my PR, sorry for that. I'll revert it if you wish so.

Reply:
👍